shared/common-adapters/reload.js

```diff
@@ -100,6 +103,14 @@ const styles = Styles.styleSheetCreate({
   },
 })
 
+type OwnProps = {|
+  children: React.Node,
+  onBack?: () => void,
```

Comment:
Should this be optional? If it has the potential to get you in a broken state on mobile maybe it should be required.

Reply:
it's optional because it doesn't apply on desktop, right?

Reply:
Yeah, but it wouldn't hurt to have it on desktop. I imagine it'll be more common for this component to live in the shared code than not, so we should make it so the types will make the component work properly in both.